sentry/resource_sentry_key.go

```diff
@@ -74,6 +80,11 @@ func resourceSentryKey() *schema.Resource {
 }
 
 func resourceSentryKeyCreate(d *schema.ResourceData, meta interface{}) error {
+	inputKeyId := d.Get("key_id").(string)
+	if inputKeyId != "" {
+		d.SetId(inputKeyId)
+		return resourceSentryKeyUpdate(d, meta)
```

Comment:
I little bit concerned about update action which is being applied while creating a key. Can we move this condition on the level upper? Something like:

```golang
    inputKeyId := d.Get("key_id").(string)
    if inputKeyId != "" {
        d.SetId(inputKeyId)
        return resourceSentryKeyUpdate(d, meta)
    } else {
        return resourceSentryKeyCreate(d, meta)
    }

```